colors/ayu.vim

```diff
@@ -323,6 +323,32 @@ call ayu#hi('xmlEqual', 'operator', '')
 
 " }}}
 
+" HTML:" {{{
+
+hi def htmlBold                term=bold cterm=bold gui=bold
+hi def htmlBoldUnderline       term=bold,underline cterm=bold,underline gui=bold,underline
+hi def htmlBoldItalic          term=bold,italic cterm=bold,italic gui=bold,italic
+hi def htmlBoldUnderlineItalic term=bold,italic,underline cterm=bold,italic,underline gui=bold,italic,underline
+hi def htmlUnderline           term=underline cterm=underline gui=underline
+hi def htmlUnderlineItalic     term=italic,underline cterm=italic,underline gui=italic,underline
+hi def htmlItalic              term=italic cterm=italic gui=italic
```

Comment:
Are those needed?
From what I can see, they're already defined.

Reply:
where's it located? when I install ayu without any other plugins, it doesn't show anything as defined by those codes. hm, maybe this is a problem with my terminal? I will check it again

Reply:
For me, it's defined here:
![image](https://user-images.githubusercontent.com/10234894/134958666-7af45754-b3a7-4e6a-b6d1-d315da1aa8cb.png)

Reply:
I agree, it looks like it needs to be removed and it is already built-in